main_routers/cloudsave_router.py

```
@@ -0,0 +1,402 @@
+            raise RuntimeError(reload_error or "reload failed")
+    except Exception as exc:
+        rollback_attempted = False
+        rollback_error = ""
+        try:
+            if backup_path:
+                rollback_attempted = True
+                restore_cloudsave_operation_backup(config_manager, backup_path)
+                initialize_character_data = get_initialize_character_data()
+                await initialize_character_data()
+                await notify_memory_server_reload(reason=f"云存档下载回滚: {name}")
+        except Exception as rollback_exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)